tox.ini

```diff
@@ -1,8 +1,16 @@
 [tox]
-envlist = pypy, pypy3, py27, py34, py35, py36
+envlist = pypy, pypy3, py27, py34, py35, py36, flake8
 
 [testenv]
 deps = -rrequirements-dev.txt
 commands =
     py.test sasstests.py
+
+[testenv:flake8]
+deps= flake8
+skip_install=True
+commands=
     flake8 .
```

Comment:
> This time savings is entirely irrelevant compared to the 10 minutes of building C bits though :)

If you like to run flake8 through tox, that's a significant downside: the `skip_install` made sure that didn't happen. But the complexity that adding all the different `[testenv:]` variants adds doesn't really warrant making people not have to set up a venv with `flake8`.